src/resolvers/subscriptions.js

```diff
@@ -30,15 +30,22 @@ const confusionAddedSubscription = {
   subscribe: withFilter(() => pubsub.asyncIterator(CONFUSION_ADDED), compareSessionId),
 }
 
+const SESSION_UPDATE = 'sessionUpdateAdded'
+const sessionUpdateSubscription = {
+  subscribe: withFilter(() => pubsub.asyncIterator(SESSION_UPDATE), compareSessionId),
+}
+
 module.exports = {
   // export the pubsub interface
   pubsub,
 
   // export subscriptions
   confusionAdded: confusionAddedSubscription,
   feedbackAdded: feedbackAddedSubscription,
+  sessionUpdateAdded: sessionUpdateSubscription,
```

Comment:
Minor remark: can you simply call this sessionUpdated, SESSION_UPDATED, sessionUpdatedSubscription, etc.?